src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import Bookreducer from './books/books';
+import categoriesReducer from './categories/categories';
+
+const rootReducer = combineReducers({
+  categories: categoriesReducer,
+  book: Bookreducer,
+});
+
+const store = configureStore({ reducer: rootReducer });
```

Comment:
[OPTIONAL] It would be better to import `applyMiddleware` to use some handy Redux middleware, like a redux logger. It will help you a lot in the development process. To install it, you need to run:

```
npm i --save redux-logger
```